src/components/htmlvideoplayer/plugin.js

```diff
@@ -580,18 +580,15 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
 
             Array.from(videoElement.textTracks)
             .filter(function(trackElement) {
-                if (customTrackIndex === -1 ) {
-                    // get showing .vtt textTacks
-                    return trackElement.mode === 'showing';
-                } else {
-                    // get current .ass textTrack
-                    return ("textTrack" + customTrackIndex) === trackElement.id;
-                }
+                // get showing .vtt textTacks
+                return (trackElement.mode === 'showing') ||
```

Comment:
Redundant parentheses and maybe consider using variables instead of cramming it all in one statement.